src/pages/Payment/PayResult.jsx

```diff
@@ -0,0 +1,39 @@
+import React, { useEffect, useState } from 'react';
+import axios from 'axios';
+import { useLocation } from 'react-router-dom';
+
+const PayResult = () => {
+  const location = useLocation();
+  const searchParams = new URLSearchParams(location.search);
```

Comment:
useSearchParams hook 활용하시면 될 것 같습니다!
```suggestion
  const [searchParams] = useSearchParams();
```

Reply:
변경했습니다!